src/api.jl

```diff
@@ -2421,16 +2421,16 @@ Get a complex vector from a [`Subscription`](@ref)
 
 - `ipt`: the [`Input`](@ref) to get the result for
 """
-function helicsInputGetComplexVector(ipt::Input)::Vector{Float64}
+function helicsInputGetComplexVector(ipt::Input)::Vector{ComplexF64}
     maxlen = Cint(helicsInputGetVectorSize(ipt))
     data = Vector{Float64}(undef, maxlen)
     actualSize = Ref(maxlen)
     @invoke_and_check Lib.helicsInputGetComplexVector(ipt, data, maxlen, actualSize)
-	complexVector = Vector{ComplexF64}(undef,0)
-	for i in 1:actualSize[]/2
-		push!(complexVector, data[2*i - 1] + im * data[2*i])
-	end
-    return compelxVector
```

Comment:
For future notice, this is where the main problem is. The rest is mostly just changing tabs to spaces.